extra/twig-extra-bundle/src/Resources/config/cache.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Cache\Adapter\TagAwareAdapter;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\Cache\TagAwareCacheInterface;
+use Twig\Extra\Cache\CacheExtension;
+use Twig\Extra\Cache\CacheRuntime;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('twig.extension.cache', CacheExtension::class)
+            ->tag('twig.extension')
+
+        ->set('twig.runtime.cache', CacheRuntime::class)
+            ->args([
+                service('twig.cache'),
+            ])
+            ->tag('twig.runtime')
+
+        ->set('twig.cache', TagAwareAdapter::class)
+            ->args([
+                service('.twig.cache.inner')
+            ])
+
+        ->set('.twig.cache.inner')
+            ->parent('cache.app')
+            ->tag('cache.pool', ['name' => 'twig.cache'])
+
+        ->alias(TagAwareCacheInterface::class.' $twigCache', 'twig.cache')
```

Comment:
Where is defined `$twigCache` ?

Reply:
It's a named autowiring alias: here, it's a string, not a variable.

Reply:
Hmm, ok, nevermind, I dot it. It's for auto wiring